specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/subAssessments.json

```diff
@@ -0,0 +1,576 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2019-01-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": ["https"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "security": [{
+    "azure_auth": ["user_impersonation"]
+  }],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{scope}/providers/Microsoft.Security/subAssessments": {

```

Comment:
If subAssessments are a child of assessments it doesn't make sense to get a collection of them outside of an individual assessment.  Looks like this is the only change from the original PR.  Is this a scenario you need to support?

Reply:
Its necessary as theres a requirement to expose them all on subscription level (mostly for scale issues on synching this data with ARG)
The parent is reflected on the object itself nevertheless , therefore we don’t loss data on this path, its only a different journey to get to it.

Reply:
Have you tried this in dogfood?  How will you represent this type in your manifest?  as assessments/subAssessments and just "subAssessments"?  Currently your manifest PR has it just listed as a "subAssessments" resourceType which seems a bit misleading.  RBAC also becomes an issue when its reprsented as two different actions.

Did ARG specifically ask for this?  Subscription level GETs are typically only used for top-level resources.